packages/electrode-archetype-react-app-dev/config/babel/babelrc-client.js

```diff
@@ -109,6 +109,13 @@ const plugins = basePlugins.concat(
 );
 
 const targets = archetype.babel.envTargets[archetype.babel.target];
+const hasOtherTargets =
+  Object.keys(archetype.babel.envTargets)
+    .sort()
+    .join(",") !== "default,node";
+const useBuiltIns = hasOtherTargets
+  ? { useBuiltIns: "entry", corejs: "2" }
```

Comment:
is it `corejs` or `core-js`?

https://babeljs.io/docs/en/babel-preset-env#usebuiltins